core/src/server/rpc_module.rs

```diff
@@ -34,7 +34,7 @@ use crate::error::{Error, SubscriptionClosed};
 use crate::id_providers::RandomIntegerIdProvider;
 use crate::server::helpers::{BoundedSubscriptions, MethodSink, SubscriptionPermit};
 use crate::server::resource_limiting::{ResourceGuard, ResourceTable, ResourceVec, Resources};
-use crate::traits::{IdProvider, ToRpcParams};
+use crate::traits::{IdProvider, ToRpcServerParams};
```

Comment:
it might be possible to remove `ToRpcServerParams` completely but let's tackle it another PR